docs/src/advanced-concepts/output-naming.rst

```diff
@@ -0,0 +1,15 @@
+Output naming
+=============
+
+The name and format of the outputs in ``metatensor-models`` are based on
+those of the ``metatensor.torch.atomistic``
+<https://lab-cosmo.github.io/metatensor/latest/atomistic/outputs.html>_
```

Comment:
The link is rendered not in way I think you expect.

https://metatensor-models--158.org.readthedocs.build/en/158/advanced-concepts/output-naming.html